internal/sbom/spdx.go

```diff
@@ -263,10 +269,39 @@ func GenerateIndexSPDX(koVersion string, sii oci.SignedImageIndex) ([]byte, erro
 
 			doc.Relationships = append(doc.Relationships, Relationship{
 				Element: ociPackageName(indexDigest),
-				Type:    "CONTAINS",
+				Type:    "VARIANT_OF",
 				Related: depID,
 			})
 
+			qual := []qualifier{{
```

Comment:
This makes me want a `newQualifier(k, v string) qualifier` helper method, and/or a `platformQualifiers(p v1.Platform) []qualifier`.

Reply:
Yeah, it's worth surveying our usage of qualifiers as this evolves and figuring out what helpers make sense.  I'd also be inclined to just create a decent OCI pURL library in GGCR and use that TBH.